app/src/main/java/com/example/habitsmasher/listeners/SwipeListener.java

```diff
@@ -0,0 +1,46 @@
+package com.example.habitsmasher.listeners;
+
+import com.example.habitsmasher.Habit;
+import com.example.habitsmasher.HabitList;
+import com.example.habitsmasher.R;
+import com.example.habitsmasher.User;
+import com.example.habitsmasher.ui.dashboard.EditHabitFragment;
+import com.example.habitsmasher.ui.dashboard.HabitItemAdapter;
+import com.example.habitsmasher.ui.dashboard.HabitListFragment;
+import com.example.habitsmasher.ui.dashboard.RecyclerTouchListener;
+/**
+ * This is the SwipeListener class
+ * It implements the RecyclerTouchListener.OnSwipeOptionsClickListener
+ * Its purpose is to perform the onSwipeOptionClicked listener actions
+ */
+public class SwipeListener implements RecyclerTouchListener.OnSwipeOptionsClickListener{
```

Comment:
From what I can see, this listener class isn't used by HabitListFragment, which uses a listener of this sort. We should probably integrate this class into HabitListFragment.

- [ ] Use SwipeListener in HabitListFragment